src/distance.jl

```diff
@@ -91,10 +91,15 @@ end
 """
     diameter(eccentricities)
     diameter(g, distmx=weights(g))
+    diameter(g::Union{SimpleGraph, SimpleDiGraph})
 
 Given a graph and optional distance matrix, or a vector of precomputed
 eccentricities, return the maximum eccentricity of the graph.
 
+An optimizied BFS algorithm (iFUB) is used for unweighted graphs, both [undirected](https://semeval.inria.fr/2012/printemps/theme1/teams/gang/62.pdf) 
```

Comment:
This links seems to be someone's personal page or some informal repository. Could you switch the link to a more official resources, something that has better chances of still existing in 20 years?